templates/monorepo/apps/azure-functions-api/.node-version

```diff
@@ -0,0 +1 @@
+18.13.0
```

Comment:
can we try with node20?

Reply:
Updated in [e7353ca](https://github.com/pagopa/dx-typescript/pull/4/commits/e7353ca236d16e983bfeff3a968afc02e0f30032)